File: commandLine.c
#include<stdio.h>
#include<stdlib.h>

int main(int argc, char *argv[])
{
	int ans=0;
	printf("Jay Ganesh\n");
	printf("Name of executable is:%s\n",argv[0]);
	printf("Number of Command line arguments are:%d\n",argc);
	
	ans= atoi(argv[1])+atoi(argv[2]);
	printf("Addition is:%d",ans);
	 return 0;
}



// gcc CommandLine.c -o myexe
//myexe 10 11

File: CommandLine.java
import java.lang.*;

class CommandLine
{
	public static void main(String Arg[])  // Arg[] is Command Line argument
	{
		System.out.println("First number is:"+Arg[0]);
		System.out.println("Second number is:"+Arg[1]);
		int ans =Integer.parseInt(Arg[0])+Integer.parseInt(Arg[1]);
		System.out.println("Addition is:"+ans);
	}
}

// javac CommandLine.java
//java CommandLine 10 11